public/js/dashboard.js

```diff
@@ -137,6 +149,7 @@ requirejs([
     Kernel.start().then(function(kernel) {
         // initialize an ipywidgets manager
         var widgetManager = new WidgetManager(kernel, _consumeMessage);
+        _handleError(kernel);
```

Comment:
Nit: name suggests to me this function is the callback. But it's really registering the callback. Maybe `_initErrorHandler`?